<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#131239;</string>
  </property>
  <widget class="QLabel" name="label_Name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>80</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Georgia</family>
     <pointsize>15</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_S_Question">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>570</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Security Question:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Con_Password">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>460</y>
     <width>291</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Username">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>160</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Password">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>360</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelHeading">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Copperplate Gothic Light</family>
     <pointsize>36</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Copperplate Gothic Light&quot;;
color:white;

</string>
   </property>
   <property name="text">
    <string>SIGNUP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_name">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="placeholderText">
    <string>Your Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_answer">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>610</y>
     <width>355</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="placeholderText">
    <string>Answer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_pass2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Confirm Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_pass1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>400</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;
</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
   <property name="placeholderText">
    <string>Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="f_uname">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>200</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="placeholderText">
    <string>Your Username</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f_create_account">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>720</y>
     <width>250</width>
     <height>61</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 16pt &quot;Arial Rounded MT Bold&quot;;
padding:15px;
background-color:green;
color:white;

</string>
   </property>
   <property name="text">
    <string>Create Account</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>680</y>
     <width>241</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Meiryo UI</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Meiryo UI&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Already have an account, </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="f_goto_login">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>680</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
     <pointsize>15</pointsize>
     <weight>9</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:transparent;
color:#0096ED;
font: 75 15pt &quot;Courier&quot;;</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_responseerror">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>500</width>
     <height>27</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Meiryo UI&quot;;
color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="f_responseerror1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>660</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Meiryo UI&quot;;
color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="f_responseerror2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>550</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Meiryo UI&quot;;
color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="f_responseerror2_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>440</y>
     <width>491</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Meiryo UI&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Hint: Password should contain atleast 8 characters,1 special character,1 uppercase character and 1 number</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QComboBox" name="f_security_ques">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>610</y>
     <width>355</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;
font: 10pt &quot;MS Shell Dlg 2&quot;;
</string>
   </property>
   <item>
    <property name="text">
     <string>Which is your favourite color?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Which is your favourite book?</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Which is your favourite subject?</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="f_email">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>400</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">padding:0px 10px 0px 10px;
border-radius:5px;
border:3px solid lightgreen;
background-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Your Email</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_Username_2">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>260</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Georgia&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>Email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_responseerror_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>500</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Meiryo UI&quot;;
color:red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label_Name</zorder>
  <zorder>label_S_Question</zorder>
  <zorder>label_Con_Password</zorder>
  <zorder>label_Username</zorder>
  <zorder>label_Password</zorder>
  <zorder>labelHeading</zorder>
  <zorder>f_name</zorder>
  <zorder>f_answer</zorder>
  <zorder>f_pass2</zorder>
  <zorder>f_pass1</zorder>
  <zorder>f_uname</zorder>
  <zorder>f_create_account</zorder>
  <zorder>label_2</zorder>
  <zorder>f_goto_login</zorder>
  <zorder>f_responseerror1</zorder>
  <zorder>f_responseerror2</zorder>
  <zorder>f_responseerror2_2</zorder>
  <zorder>f_security_ques</zorder>
  <zorder>label_Username_2</zorder>
  <zorder>f_email</zorder>
  <zorder>f_responseerror_2</zorder>
  <zorder>f_responseerror</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
